src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -135,6 +141,17 @@ const RichTextPopup = () => {
   const syncDraggablePosition = (e, { x, y }) => {
     setDraggablePosition({ x, y });
   };
+
+
+  const insertSymbols = symbol => {
+    const contents = editorRef.current.getContents();
+    const { index, length } = editorRef.current.editor.getSelection();
+    // only update the selected word (defined by index)
+    editorRef.current.editor.setText(`${contents.slice(0, index)}${symbol}${contents.slice(index + length)}`);
+    editorRef.current.editor.setSelection(index+1);

```

Comment:
`setText` and `setSelection` will be gone in the production build as they are not "exposed". More details can be found at `RichTextEditor.ts`.

For `setSelection` we can do `current.setSelection` instead of `current.editor.setSelection`.
For `setText` we would want to expose a wrapper method in the `RichTextEditor.ts`. For more details please take a look at other exposed methods in the file.

Reply:
Thanks for teaching.
I update Webviewer to expose a wrapper method `setText`, see PR:  https://github.com/XodoDocs/webviewer/pull/1599
Also I changed `current.editor.setSelection` to `current.setSelection` and `current.editor.setText` to `current.setText`